test/tests/functional/pbs_job_array.py

```diff
@@ -659,3 +659,25 @@ def test_subjob_stdfile_custom_dir(self):
                 if f_name not in file_list:
                     raise self.failureException("std file " + f_name
                                                 + " not found")
+
+    def test_subjob_wrong_state(self):
+        """
+        Test that after submitting a job and restarting the server,
+        the subjobs are not in the wrong substate and can be scheduled
+        """
+        a = {'resources_available.ncpus': 500}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        a = {ATTR_J: '1-500'}
+        j = Job(attrs=a)
```

Comment:
Consider putting the dictionary directly into Job() instead of assigning it to a.  It won't go over the 79 char limit.